pkg/agent/storage/stores.go

```diff
@@ -15,6 +15,7 @@ const (
 	StatusLogsStore             Store = "status_logs"              // The store used for buffered status logs.
 	ServerProvidedDataStore     Store = "server_provided_data"     // The store used for pushing values from server-backed tables.
 	TokenStore                  Store = "token_store"              // The store used for holding bearer auth tokens, e.g. the ones used to authenticate with the observability ingest server.
+	ControlServerActionsStore   Store = "action_store"             // The store used for storing actions sent by control server.
```

Comment:
so this will take the place of the sent notifications store which means we update to this it will be a fresh store and we will possibly send old notifications form K2.

should we ...

1. add a migration that transfers the sent notifications to actions store
2. have action queue look in both stores temporarily
3. something else ...

Reply:
went with 2!